src/Validator.ts

```diff
@@ -333,11 +333,14 @@ export class Validator implements IValidator {
   }
 
   verifyAMTVersion (payload: Payload, action: string): void {
-    if (parseInt(payload.ver) >= 7 && parseInt(payload.ver) < 12) {
+    // firmware versions 6.x, 7.x, 8.x 9.x, 10.x, 11.0, 11.5, and 11.6 that can allow an unprivileged attacker to gain control of the manageability features
+    // https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00075.html
+    const vulnerableVersions = [7.0, 7.1, 8.0, 8.1, 9.0, 9.1, 9.5, 10.0, 11.0, 11.5, 11.6]
```

Comment:
Can we change the variable name to verifiedVersions or something?  I don't like having "vulnerable" in there. Also, lets remove the comments above.

Reply:
done